src/test/github/pullRequestModel.test.ts

```diff
@@ -132,12 +132,12 @@ const pr: Octokit.PullRequestsGetResponse | Octokit.PullRequestsGetAllResponseIt
 	milestone: null,
 	locked: false,
 	active_lock_reason: '',
-	closed_at: null,
-	merged_at: null,
-	_links: null,
-	merge_commit_sha: null,
+	closed_at: undefined,
+	merged_at: undefined,
+	_links: undefined,
+	merge_commit_sha: undefined,
 	mergeable: true,
-	merged_by: null,
+	merged_by: undefined,
```

Comment:
These lines fix some `tsc` warnings about `null` not being compatible with `string | undefined`.